apps/site/open-next.config.ts

```diff
@@ -1,8 +1,16 @@
 import type { OpenNextConfig } from '@opennextjs/cloudflare';
 import { defineCloudflareConfig } from '@opennextjs/cloudflare';
-import incrementalCache from '@opennextjs/cloudflare/overrides/incremental-cache/kv-incremental-cache';
+import r2IncrementalCache from '@opennextjs/cloudflare/overrides/incremental-cache/r2-incremental-cache';
+import { withRegionalCache } from '@opennextjs/cloudflare/overrides/incremental-cache/regional-cache';
+import doQueue from '@opennextjs/cloudflare/overrides/queue/do-queue';
 
-const cloudflareConfig = defineCloudflareConfig({ incrementalCache });
+const cloudflareConfig = defineCloudflareConfig({
+  incrementalCache: withRegionalCache(r2IncrementalCache, {
```

Comment:
Can you add inline comments reference `@see` to relevant docs? And also inline comment what these settings do? 🙇

Reply:
The PR description mentions

> replaced the kv incremental cache with the more performance r2 + regional cache implementation

The actual issue with KV is that it is eventually consistent which might lead to unexpected behavior. This is why we recommend R2 instead - R2 is strongly consistent.

Reply:
@ovflowd I've added an inline comment https://github.com/nodejs/nodejs.org/pull/8304/commits/65b4b4b39306e307764a4e1afb986bbbabe3d122

please let me know if this works for you 🙏